packages/plugins/documentation/admin/src/pages/PluginPage/tests/index.test.js

```diff
@@ -104,4 +115,22 @@ describe('Plugin | Documentation | PluginPage', () => {
       </div>
     `);
   });
+
+  it('should show a loader when fetching data', () => {
+    const history = createMemoryHistory();
+    const App = makeApp(history);
+    render(App);
+
+    expect(screen.getByTestId('loader')).toBeInTheDocument();
+  });
+
+  it('should show a list of versions', async () => {
+    const history = createMemoryHistory();
+    const App = makeApp(history);
+    render(App);
+
+    await waitFor(() => {
+      expect(screen.getByText('1.0.0')).toBeInTheDocument();
+    });
```

Comment:
```suggestion
    await waitFor(() => 
      expect(screen.getByText('1.0.0')).toBeInTheDocument()
    );
```

waitFor needs to receive a promise. In your case, it returns something that always resolves undefined. Removing the extra `{` is probably more consistent in this scenario :)